lib/warder/version.rb

```diff
@@ -1,4 +1,4 @@
 # defines warder version
 module Warder
-  VERSION = '0.2.4'
+  VERSION = '0.3.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.